File: ios/oplayer/Model/GatewayAssetItemData.m
//
//  GatewayAssetItemData.m
//  oplayer
//
//  Created by SYALON on 13-11-20.
//
//

#import "GatewayAssetItemData.h"
#import "OrgUtils.h"

@interface GatewayAssetItemData()
{
}

@end

@implementation GatewayAssetItemData

@synthesize enableDeposit, enableWithdraw;
@synthesize symbol, backSymbol, name;

@synthesize intermediateAccount, balance;

@synthesize depositMinAmount, withdrawMinAmount, withdrawGateFee;
@synthesize supportMemo;
@synthesize confirm_block_number;
@synthesize coinType, backingCoinType;
@synthesize depositMaxAmountOnce, depositMaxAmount24Hours, withdrawMaxAmountOnce, withdrawMaxAmount24Hours;

@synthesize gdex_backingCoinItem;
@synthesize open_withdraw_item, open_deposit_item;

- (void)dealloc
{
}

@end

File: android/app/src/main/java/com/btsplusplus/fowallet/gateway/GatewayAssetItemData.kt
package com.btsplusplus.fowallet.gateway

import org.json.JSONObject

class GatewayAssetItemData {

    public var enableDeposit: Boolean = false
    public var enableWithdraw: Boolean = false

    public var symbol: String = ""
    public var backSymbol: String = ""
    public var name: String = ""

    public var intermediateAccount: String? = null
    public lateinit var balance: JSONObject

    public var depositMinAmount: String? = null
    public var withdrawMinAmount: String? = null
    public var withdrawGateFee: String? = null

    public var supportMemo: Boolean = false
    public var confirm_block_number: String? = null

    public var coinType: String = ""
    public var backingCoinType: String = ""

    public var depositMaxAmountOnce: String? = null
    public var depositMaxAmount24Hours: String? = null
    public var withdrawMaxAmountOnce: String? = null
    public var withdrawMaxAmount24Hours: String? = null

    public var gdex_backingCoinItem: JSONObject? = null
    public var open_withdraw_item: JSONObject? = null
    public var open_deposit_item: JSONObject? = null

}


